local/build-binary

```diff
@@ -36,6 +36,37 @@ mkdir -p ${BUILD_DIR} ${INSTALL_DIR} ${DOWNLOAD_DIR}
 SOURCEDIR=$(cd ${2:-$(dirname $0)/../sources}; pwd -P)
 
 
+# ------------------------------------------------------------------------------
+# Setup ccache if enabled
+# ------------------------------------------------------------------------------
+if [[ "${USE_CCACHE}" == "1" ]]; then
+    echo "Setting up ccache..."
+
+    # Configure ccache (already installed in Docker image)
+    if [[ -n $(command -v ccache) ]]; then
+        ccache --version
+        ccache --max-size=8G
```

Comment:
We can't change `ccache` settings in `local/build-binary` because when somebody will run this script locally it will change his/her `ccache`. I have `max_size = 256.0 GB` for example.